packages/docs/pages/select.tsx

```diff
@@ -1,399 +1,499 @@
+                  <CodePreview>
+                    {/* jsx-to-string:start */}
+                    <Grid gridColumns="repeat(4, 1fr)">
+                      <Select
+                        label="Select"
+                        onOptionChange={() => null}
+                        options={[
+                          { value: 1, content: 'Option' },
+                          { value: 2, content: 'Option' },
+                          { value: 3, content: 'Option' },
+                          { value: 4, content: 'Option' },
+                        ]}
+                        placeholder="Choose from above"
+                        placement="top"
+                        required
+                      />
+                      <Select
+                        label="Select"
+                        onOptionChange={() => null}
+                        options={[
+                          { value: 1, content: 'Option' },
+                          { value: 2, content: 'Option' },
+                          { value: 3, content: 'Option' },
+                          { value: 4, content: 'Option' },
+                        ]}
+                        placeholder="Choose from below"
+                        placement="bottom-start"
+                        required
+                      />
+                      <Select
+                        label="Select"
+                        onOptionChange={() => null}
+                        options={[
+                          { value: 1, content: 'Option' },
+                          { value: 2, content: 'Option' },
+                          { value: 3, content: 'Option' },
+                          { value: 4, content: 'Option' },
+                        ]}
+                        placeholder="Choose from the right"
+                        placement="right-start"
+                        required
+                      />
+                      <Select
+                        label="Select"
+                        onOptionChange={() => null}
+                        options={[
+                          { value: 1, content: 'Option' },
+                          { value: 2, content: 'Option' },
+                          { value: 3, content: 'Option' },
+                          { value: 4, content: 'Option' },
+                        ]}
+                        placeholder="Choose from the left"
+                        placement="left-end"
+                        required
+                      />
+                    </Grid>
+                    {/* jsx-to-string:end */}
+                  </CodePreview>
+                </Fragment>
+              ),
+            },
+            {
+              id: 'max-height',
+              title: 'Max height',
+              render: () => (
+                <Fragment key="max-height">
+                  <Text>
+                    Once the content is longer than the max-height, the <Code primary>Select</Code> will be scrollable.
```

Comment:
I updated `the Dropdown will be scrollable` to `the Select will be scrollable`? Is that ok? other suggestions?

Reply:
Good to me.